app/controllers/api/v2/courses_controller.rb

```diff
@@ -0,0 +1,12 @@
+module API
+  module V2
+    class CoursesController < ApplicationController
+      def index
+        provider = Provider.find_by!(provider_code: params[:provider_code])
```

Comment:
We should be getting Pundit or similar in here to handle authorization.

Reply:
Happy to do this as a later PR, however.

Reply:
this thing, https://github.com/varvet/pundit 

ok